CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 project(ZMAP C)
-set(ZMAP_VERSION 2.1.1)
+set(ZMAP_VERSION DEVELOPMENT) # Change DEVELOPMENT to version number for release
```

Comment:
More context: I think the plan was that release versions would be git branches with the variable replaced with the release number. That way even when building from source you'd get the version rather than the dev string. Of course this could also be done by the branch setting the dev mode 'off' by default.